src/views/Pools/components/BountyModal.tsx

```diff
@@ -28,23 +25,32 @@ const Divider = styled.div`
   width: 100%;
 `
 
-const BountyModal: React.FC<BountyModalProps> = ({
-  cakeBountyToDisplay,
-  dollarBountyToDisplay,
-  totalPendingCakeHarvest,
-  callFee,
-  onDismiss,
-  TooltipComponent,
-}) => {
+const BountyModal: React.FC<BountyModalProps> = ({ onDismiss, TooltipComponent }) => {
   const { t } = useTranslation()
   const { account } = useWeb3React()
   const { theme } = useTheme()
   const { toastError, toastSuccess } = useToast()
   const cakeVaultContract = useCakeVaultContract()
-  const [pendingTx, setPendingTx] = useState(false)
+  const [pendingTx, setPendingTx] = useState(true)
```

Comment:
Should this now be defaulting to true?

Reply:
Sorry it was a left over from testing, fixed now :)